### PR TITLE
doc: release-notes 2024-10-03

### PR DESCRIPTION
**Pull request by nikromen**

### CODE REVIEW

File: doc/release-notes/2024-10-03.rst [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+==================================
+
+Enhancements
+------------
+
+- Dist git client moved from Copr to DistGit `PR#3299`_.
+- Jakub started to adding support for Pulp in Copr. We are slowly adding more
+  features to it. Tracking epic issue is `Issue#2533`_.
+- Copr has a nice view of webhook history for each project. You can see what
+  webhooks were sent and from where `PR#3342`_ and `PR#3407`_.
```
Comment (xsuchy) [inline doc/release-notes/2024-10-03.rst:13]: Maybe mention where users can find the information.

File: doc/release-notes/2024-10-03.rst [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+- Do not depend on `yum` for Fedora 41 and later inside copr-rpmbuild `PR#3395`_.
+- Support python 3.14 version `PR#3363`_.
+- Fail-safe build timeout added in case copr-rpmbuild is stuck `PR#3361`_.
+- Fix URL redirect for centos-stream `PR#3341`_.
+
+
+Warnings/Future changes
+-----------------------
+
+- Custom method timeout was incriased from 60 minutes to 90 minutes `PR#3404`_.
```
Comment (xsuchy) [inline doc/release-notes/2024-10-03.rst:36]: typo in "Incriased"

File: doc/release-notes/2024-10-03.rst (hunk 1 above)
Comment (FrostyX) [reply doc/release-notes/2024-10-03.rst:13]: Also, I'd like to explicitly mention that the feature was contributed from the community as a response to https://frostyx.cz/posts/do-you-want-to-contribute-to-copr

File: doc/release-notes/2024-10-03.rst [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+Bugfixes
+--------
+
+- Add missing diff.txt file for fedora-review tool `PR#3304`_
+- Fixing the 500 error - race condition when creating a new project multiple
+  times `PR#3415`_.
+- Do not depend on `yum` for Fedora 41 and later inside copr-rpmbuild `PR#3395`_.
+- Support python 3.14 version `PR#3363`_.
+- Fail-safe build timeout added in case copr-rpmbuild is stuck `PR#3361`_.
```
Comment (praiskup) [inline doc/release-notes/2024-10-03.rst:31]: nit: s/build/source build/

**Comment by praiskup** [comment]
Can you update the `docs/release_notes.rst` file?

**Comment by praiskup** [comment]
Otherwise LGTM

**Pull request merged**